### ISSUE TITLE
Almost got this running...

### ISSUE THREAD

**Issue by Xper1** [openeventdata/mordecai#51]
Hi, Thank you for all of your help and my apologies for the string of questions but I'm very new to programing. I get the following message when I run my request. Does the error message indicate anything to you that I might be doing wrong? 

Thank you again, 

![screen shot 2018-05-24 at 9 32 09 pm](https://user-images.githubusercontent.com/39590005/40510567-689ebb7e-5f9d-11e8-968e-0d0228527b53.png)

**Comment by ahalterman** [comment]
The error message says that Mordecai couldn't reach Elasticsearch on port 9200. You should make sure it's correctly running there. Did you run the setup lines in the readme? Did you see any errors then?

What do you see when you run `docker ps`? Or if you run `docker ps -a` do you see the shut down container? You can use `docker logs` to try to figure out what's going on.

**Comment by Xper1** [comment]
Ok, I got it running. It looks like I had to do three things: First, run docker, second, make sure that the elasticsearch-2.4.6 and geonames_index folders were in the same folder as the python script and the document I want to run the script on, and third, type "docker run -d -p 127.0.0.1:9200:9200 -v $(pwd)/geonames_index/:/usr/share/elasticsearch/data elasticsearch:5.5.2" into the terminal. After this I was able to run the python script with Mordecai. 

Thank you again for all of your help. I really appreciate it!
